src/index.html

```diff
@@ -1,13 +1,655 @@
-    <title>Dia</title>
+    <title>Air</title>
     <link rel="stylesheet" href="./styles/main.scss">
-  </head>
-  <body>
-    <h1>Dia</h1>
     <script type="text/javascript" src="scripts/main.js"></script>
+    <script type="module" src="scripts/slider.js"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600;700&family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=""
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      type="Image/png"
+      href="./images/favicon.png"
+      rel="icon"
+    >
+  </head>
+  <body class="page__body">
+    <header class="page__header header">
+      <div class="container">
+        <nav class="header__nav nav">
+
+          <a href="#" class="nav__logo logo">
+            <img
+              src="./images/Air.svg"
+              alt="air"
+              class="logo__image"
+            >
+          </a>
+
+          <a href="#menu" class="nav__menu-opener"></a>
+
+          <ul class="nav__list" id="nav">
+            <li class="nav__item">
+              <a href="#about-as" class="nav__link link">About us</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#services" class="nav__link link">Services</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#testimonials" class="nav__link link">Testimonials</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#contact-us" class="nav__link link">Contact Us</a>
+            </li>
+          </ul>
+
+          <a href="#contact-us" class="nav__hire-us">
+            Hire us
+          </a>
+        </nav>
+      </div>
+    </header>
+
+    <section class="page__section strategic">
+      <section class="strategic__content">
+        <h1 class="strategic__title">
+          Strategic Agency
+        </h1>
+
+        <p class="strategic__info">
+          We believe in the power of bold ideas that can solve business challenges.
+        </p>
+
+        <a
+          href="#expertise"
+          class="strategic__button button button__link"
+        >
+          Learn more
+        </a>
+      </section>
+
+      <div class="strtegic__intro intro">
+        <div class="intro__pictures">
+          <img
+            src="./images/watchbig.jpg"
+            alt=""
+            class="intro__slider"
+          >
+
+          <img
+          src="./images/slider/headphones.jpg"
+          alt=""
+          class="intro__slider"
+          >
```

Comment:
Fix code style on all code

```suggestion
          <img
            src="./images/slider/headphones.jpg"
            alt=""
            class="intro__slider"
          >
```